roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -100,6 +100,16 @@ spec:
 {% if kube_feature_gates %}
     - --feature-gates={{ kube_feature_gates|join(',') }}
 {% endif %}
+{% if kube_version | version_compare('1.9', '>=') %}
+    - --requestheader-client-ca-file=/etc/kubernetes/ssl/ca.pem
+    - --requestheader-allowed-names=system:aggregator-proxy-client
```

Comment:
Create a new variable in roles/kubernetes/master/defaults/main.yml:
`kube_api_requestheader_allowed_names: "system:aggregator-proxy-client"`
^ needs to be quoted in yml file because it contains ":"
Change this line to:
`    - --requestheader-allowed-names={{ kube_api_requestheader_allowed_names }}`

Reply:
Actually, please change it so we can make this equivalent with what is in kubeadm:
https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.8.md

The name should be just `front-proxy-client`